modules/logging/logging.go

```diff
@@ -37,31 +39,38 @@ func SharedInstance() *LogStruct {
 	return sharedInstance
 }
 
-func (u *LogStruct) MethodInfo(model string, method string, stack ...bool) *logrus.Entry {
-	if len(stack) > 0 && stack[0] {
+func (u *LogStruct) MethodInfo(model string, method string, stack bool, context ...web.C) *logrus.Entry {
```

Comment:
__[golint]__
* [golint] exported method LogStruct.MethodInfo should have comment or be unexported